### PR TITLE
enh: Stable resource access fix

### LINKED ISSUE #5674: Replace all 3rd party links to reliable resource
**Describe the bug**
Many links which have either been used for testing/mainstream access  using `urllib` are often blocked due to some issue with the site security or some other reason. 

**Expected behavior**
All links must point to a certain reliable resource like FOSSASIA's website to ensure reliable access.

**Stacktrace**
One of the plausible errors which might occur `urllib.error.HTTPError: HTTP Error 403: Forbidden`

**Additional Info**:
Working  on it

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #5674 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
To ensure reliability, replaces third party links to FOSSASIA resources

#### Changes proposed in this pull request:

- resource provider method for image and static links

### CODE REVIEW

**Comment by mrsaicharan1** [comment]
@iamareebjamal  Is this fine?

File: app/api/helpers/utilities.py [hunk 1]
```diff
@@ -113,6 +113,10 @@ def update_state(task_handle, state, result=None):
         )
 
 
+def resource_links():
```
Comment (iamareebjamal) [inline app/api/helpers/utilities.py]: Why create a function, just create variables

**Comment by mrsaicharan1** [comment]
Yes will do that.

File: app/api/helpers/utilities.py [hunk 2]
```diff
@@ -112,6 +112,8 @@ def update_state(task_handle, state, result=None):
             state=state, meta=result
         )
 
+resource_links = {'static_page': 'https://eventyay.com/',
```
[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/api/helpers/utilities.py]: No need for a dictionary, just create 2 variables

**Comment by mrsaicharan1** [comment]
@iamareebjamal Please review

**Pull request merged**